trieste/models/keras/architectures.py

```diff
@@ -142,6 +158,35 @@ def connect_layers(self) -> tuple[tf.Tensor, tf.Tensor]:
         raise NotImplementedError
 
 
+class MultivariateNormalTriL(tfp.layers.MultivariateNormalTriL):  # type: ignore[misc]
+    """Fixed version of tfp.layers.MultivariateNormalTriL that handles saving."""
```

Comment:
can you add a link to the issue so we keep track of it? or perhaps some dummy test that will be triggered?

Reply:
that would probably give us a boilerplate test to check whether a layer can be deepcopied as well

Reply:
(I don't understand. Some tfp layers are savable, others aren't. MNTL wasn't savable so I fixed it so it is.)

Reply:
it seems useful to have a test that when deepcopying the original MNTL we get a failure - if after upgrading to some future TFP version this will have been resolved, such test would fail and alert us to remove our customization, no?

Reply:
gotcha

Reply:
added